WriteHeader/calibration.py

```diff
@@ -0,0 +1,23 @@
+import os
+from numpy import matrix as M
+from calibration_tool import CalibrationTool
+
+basedir = 'C:\Users\chris\Desktop\Calibration\CadaverData'
+cam1_path = '\Cam1'
+cam2_path = '\Cam2'
+calibration_folder_path = '\Calibration'
+file_name_suffix = '_cd001_NM_Calib_UP_0001UNDIST_JTCalibration.txt'
+output_path = 'OUTPUT'
+
+camera_1 = CalibrationTool()
```

Comment:
Interface for CalibrationTool is strange. I would typically expect one call for one calibration. A calibration is defined for both the 2D images and the 3D image, no?